### PR TITLE
update lit to 3.0.0

### PR DESCRIPTION
**Pull request by ZheSun88**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vaadin/bundles",
-  "version": "24.3.0-alpha4",
+  "version": "24.3.0-alpha5",
```
Comment (web-padawan) [inline package.json:3]: This needs to be changed back to `alpha4`.

Comment (ZheSun88) [reply package.json:3]: okay

Comment (ZheSun88) [reply package.json]: updated.

Comment (ZheSun88) [reply package.json]: oops, test failed.. seems this version should be the same with the component version

**Review by web-padawan — APPROVED** [review]

**Pull request merged**